src/images/index.js

```diff
@@ -97,7 +97,7 @@ export sending_50_circle_big from './sending-50-circle-big.png'
 export sending_50_circle_small from './sending-50-circle-small.png'
 export sending_75_circle_big from './sending-75-circle-big.png'
 export sending_75_circle_small from './sending-75-circle-small.png'
-export spinner_wheel_svg from './spinner-wheel.svg'
+// export spinner_wheel_svg from './spinner-wheel.svg'
```

Comment:
Should we uncomment this line?

Reply:
Nope. We can't import svg files directly. Usage example may be found here: https://github.com/vault-development/react-native-svg-uri